exp/test_cases/MiMA/#MiMA_test_case.py#

```diff
@@ -0,0 +1,179 @@
+import os
```

Comment:
What is this file @ntlewis?  Is it a temp made by emacs or vim?  If not, could you rename it?

Reply:
Will remove.